docs/cli-ref.rst

```diff
@@ -35,3 +35,48 @@ development and testing.
    :module: airflow.cli.cli_parser
    :func: get_parser
    :prog: airflow
+
+Environment Variables
+'''''''''''''''''''''
+
+.. envvar:: AIRFLOW__{SECTION}__{KEY}
+
+  Sets options in the Airflow configuration. This takes priority over the value in the ``airflow.cfg`` file.
+
+  Replace the ``{SECTION}`` placeholder with any section
+  and the ``{KEY}`` placeholder with any key in that specified section.
+
+  For example, if you want to set the ``dags_folder`` options in ``[core]`` section,
+  then you should set the ``AIRFLOW__CORE__DAGS_FOLDER`` environment variable.
+
+  For more information, see: :doc:`/howto/set-config`.
+
+.. envvar:: AIRFLOW__{SECTION}__{KEY}_CMD
+
+  For any specific key in a section in Airflow,
```

Comment:
This section does not say that this is used to set configuration options. It only says that the command is run, but not why it is run. It is worth writing that the command is run and the result is used as a value for the configuration option.